web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -2167,4 +2171,92 @@ describe("<BusinessFormationPaginator />", () => {
       page.fillText("Search business name", "Pizza Joint");
     };
   });
+
+  describe("remembers formation step", () => {
+    it("shows the Business Step with an initial user when lastVisitedPage has index 0 which is by default", async () => {
+      preparePage(initialUserData, displayContent);
+      expect(screen.getByTestId("business-name-step")).toBeInTheDocument();
+      await waitFor(() => {
+        expect(currentUserData().formationData.lastVisitedPageIndex).toEqual(0);
+      });
+    });
+
+    it("shows the last formation step when lastVisitedPage has step index 4", () => {
+      const legalStructureId = "limited-liability-company";
+      const profileData = generateFormationProfileData({ legalStructureId });
+      const formationData = generateFormationData({ lastVisitedPageIndex: 4 });
+
+      initialUserData = generateUserData({ profileData, formationData });
+      preparePage(initialUserData, displayContent);
+      expect(screen.getByTestId("review-step")).toBeInTheDocument();
+    });
+
+    it("formState.lastVistedPage updates on step change via stepper", async () => {
```

Comment:
Should it be renamed to say userData instead of formState because we are checking the currentUserData in the test and we don't refer to form state or the state in any of our tests so it might confusing down the line?

Reply:
Good catch and yes you're totally right, that was an artifact from earlier